src/test/ui/rust-2018/uniform-paths/redundant.stderr

```diff
@@ -0,0 +1,14 @@
+error: `std` import is redundant
+  --> $DIR/redundant.rs:15:5
+   |
+LL | use std;
+   |     ^^^
+   |     |
+   |     refers to external crate `::std`
+   |     defines `self::std`, shadowing itself
+   |
+   = help: remove or write `::std` explicitly instead
```

Comment:
Wouldn't `use ::someexterncrate;` be redundant as well, because `extern_prelude` makes `somecrate` already available?